<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPololuMaestroTester</class>
 <widget class="QDialog" name="DlgPololuMaestroTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pololu Maestro Tester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>port:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>channel/ID:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>set value:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>read value:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>messages:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labMessage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>speed:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbBaudrate">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>connect</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QWidget" name="w3" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinChannel"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QWidget" name="w1" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="maximum">
           <number>4080</number>
          </property>
          <property name="value">
           <number>1500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSet">
          <property name="suffix">
           <string> µs</string>
          </property>
          <property name="maximum">
           <number>4080</number>
          </property>
          <property name="value">
           <number>1500</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QWidget" name="w2" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>4080</number>
          </property>
          <property name="value">
           <number>64</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinRead">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> µs</string>
          </property>
          <property name="maximum">
           <number>4080</number>
          </property>
          <property name="value">
           <number>64</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbPort">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DlgPololuMaestroTester</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>w3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>w2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>toggled(bool)</signal>
   <receiver>w1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinSet</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinSet</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRead</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinRead</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
